dissect/target/plugins/os/windows/dpapi/dpapi.py

```diff
@@ -5,13 +5,28 @@
 
 from Crypto.Cipher import AES
 
-from dissect.target import Target
 from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers import keychain
 from dissect.target.plugin import InternalPlugin
 from dissect.target.plugins.os.windows.dpapi.blob import Blob as DPAPIBlob
 from dissect.target.plugins.os.windows.dpapi.master_key import CredSystem, MasterKeyFile
 
 
+def get_passwords() -> set:
+    keys = (
+        keychain.get_keys_for_provider("dpapi")
+        + keychain.get_keys_for_provider("browser")
```

Comment:
I still haven't read up on Chrome password lore so still going off of this code alone, but I assume these are user logon passwords? Wouldn't it make more sense to only use a single `user` provider within DPAPI?